index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
@nmwd  I'm trying to figure out if this file should actually be here.  If this is going to get cloned into any project, this file will most likely be deleted right away.  If we do need it, it should be renamed, or maybe http://documentcss.com/ would help solve this?